include/robot_localization/time_to_publish.h

```diff
@@ -93,6 +97,16 @@ namespace RobotLocalization
   {
   }
 
+  TimeToPublish::~TimeToPublish()
+  {
+    // Stop timing thread if running
+    if (thread_)
```

Comment:
I think this is the important part.

I'm not 100% sure of using a RAW pointer for the condition_variable, but it seems that ownership is handled from outside of this class, right?